### ISSUE TITLE
When training other language, is text token prediction in loss calculation beneficial for language understanding?

### ISSUE THREAD

**Issue by hopingZ** [knottwill/sesame-finetune#4]
Thanks for your great work! I would like to know that when training a new language, such as Chinese, which is a language quite different from English, is it necessary to include text token prediction in the loss calculation to enhance the language understanding ability for Chinese? What's your opinion on this?

**Comment by knottwill** [comment]
Admittedly, I have only trained on french, spanish, german, and swedish data so far, which do not seem to need any text prediction loss. I would highlight that as a TTS model, it is very unlikely that the pretraining involved predicting text. If you decide to try that out though, I would be very interested to hear about your results.

**Comment by NHDat2** [comment]
hey @knottwill , first of all, Thank u for your work.
I have 2 ques, I hope to get answer from u
1. this code will work with french, right ? 
2. how many data for each language which u guy used ?
